pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java

```diff
@@ -360,10 +364,19 @@ private FilterManager createFilterManager(
       final FilterManager filterManager,
       final Optional<AccountWhitelistController> accountWhitelistController,
       final PrivacyParameters privacyParameters) {
+    JsonNode genesisConfigConfig;
+    try {
+      genesisConfigConfig =
+          new ObjectMapper().readTree(ethNetworkConfig.getGenesisConfig()).get("config");
+    } catch (final IOException ioe) {
+      genesisConfigConfig = null;
+    }
```

Comment:
Definitely not keen on having to parse the JSON a second time here (as opposed to just in `GenesisConfigFile.fromConfig`).

Reply:
The json de/serialization will address this.